docs/components/zeebe/technical-concepts/internal-processing.md

```diff
@@ -57,16 +57,15 @@ This command can in turn be processed, completing the service task and driving t
 
 ## Handling backpressure
 
-When a broker receives a client request, it is written to the **event stream** first, and processed later by the stream processor. If the processing is slow or if there are many client requests in the stream, it might take too long for the processor to start processing the command. If the broker keeps accepting new requests from the client, the back log increases and the processing latency can grow beyond an acceptable time.
+When a broker receives a client request, it is written to the **event stream** first, and processed later by the stream processor. If the processing is slow or if there are many client requests in the stream, it might take too long for the processor to start processing the command. If the broker keeps accepting new requests from the client, the backlog increases and the processing latency can grow beyond an acceptable time.
 
 To avoid such problems, Zeebe employs a [backpressure](/self-managed/zeebe-deployment/operations/backpressure.md) mechanism.
 When the broker receives more requests than it can process with an acceptable latency, it rejects some requests.
 
-The maximum rate of requests that can be processed by a broker depends on the processing capacity of the machine, the network latency, current load of the system, etc.
+This is indicated to the client by a **resource exhausted** exception. If a client sees this exception, it can retry the requests with an appropriate retry strategy. If the rejection rate is high, it indicates the broker is constantly under high load and you need to reduce the rate of requests. Alternatively, you can also increase broker resources to adjust to your needs. In high-load scenarios, it is recommended to [benchmark](https://camunda.com/blog/2022/05/how-to-benchmark-your-camunda-platform-8-cluster/) your Zeebe broker up front to size it correctly.
```

Comment:
What is "This" in "this is indicated by..."? Do you mean "These rejected requests are indicated to the client by a..." or "This backpressure mechanism is indicated to the client by a..."?

Reply:
"Backpressure is indicated"